NuRadioReco/modules/io/RNO_G/readRNOGDataMattak.py

```diff
@@ -57,8 +73,12 @@ def baseline_correction(wfs, n_bins=128, func=np.median):
     # np.moveaxis -> (n_events, n_channels, 2048)
     baseline_traces = np.moveaxis(baseline_traces, 0, -1)
      
+    if return_offsets:
+        return wfs - baseline_traces, baseline_values
+    
     return wfs - baseline_traces
 
+blockoffsetfitter = channelBlockOffsets()
```

Comment:
That that here in between functions

Reply:
I've made it a private attribute of the `readRNOGData` class, having `readRNOGDataMattak.blockoffsetfitter` showing up in IDE autocompletion is probably unnecessary clutter.